CHANGELOG.md

```diff
@@ -3,71 +3,129 @@
-* Build: change --console=[auto,false,true] to --progress=[auto,plain,tty]. [docker/cli#1276](https://github.com/docker/cli/pull/1276)
-* Builder: Implement `builder prune` to prune build cache. [docker/cli#1295](https://github.com/docker/cli/pull/1295)
-* Builder: do not cancel buildkit status request. [moby/moby#37597](https://github.com/moby/moby/pull/37597)
-* Builder: implement PullParent option with buildkit. [moby/moby#37613](https://github.com/moby/moby/pull/37613)
-* Buildkit: enable net modes and bridge. [moby/moby#37620](https://github.com/moby/moby/pull/37620)
-* Import TestBuildIidFileSquash from moby to cli. [docker/cli#990](https://github.com/docker/cli/pull/990)
-* Re-add support for a custom .bashrc file in build env. [moby/moby#37370](https://github.com/moby/moby/pull/37370)
-* Remove "experimental" annotations for buildkit. [docker/cli#1303](https://github.com/docker/cli/pull/1303)
-* Remove experimental guard for buildkit. [moby/moby#37686](https://github.com/moby/moby/pull/37686)
-* Set BuildKit's ExportedProduct variable to show useful errors in the future. [moby/moby#37439](https://github.com/moby/moby/pull/37439)
-* Vendor: update buildkit to 9acf51e491. [moby/moby#37385](https://github.com/moby/moby/pull/37385)
-* Vndr buildkit, containerd, console. [docker/cli#1302](https://github.com/docker/cli/pull/1302)
-* Vndr containerd to a88b631, buildkit to e57eed4, and fsutil to b19464c. [moby/moby#37582](https://github.com/moby/moby/pull/37582)
-* [enhancement] add optional fields in daemon.json to enable buildkit. [moby/moby#37593](https://github.com/moby/moby/pull/37593)
-* [enhancement] enable buildkit from daemon side. [docker/cli#1275](https://github.com/docker/cli/pull/1275)
-+ Add osusergo build tar for static binaries. [moby/moby#37500](https://github.com/moby/moby/pull/37500)
++ Allow BuildKit builds to run without experimental mode enabled. Buildkit can now be configured with an option in daemon.json [moby/moby#37593](https://github.com/moby/moby/pull/37593) [moby/moby#37686](https://github.com/moby/moby/pull/37686) [moby/moby#37692](https://github.com/moby/moby/pull/37692) [docker/cli#1303](https://github.com/docker/cli/pull/1303)  [docker/cli#1275](https://github.com/docker/cli/pull/1275)
++ Add support for build-time secrets using a `--secret` flag when using BuildKit [docker/cli#1288](https://github.com/docker/cli/pull/1288)
++ Add SSH agent socket forwarder (`docker build --ssh $SSHMOUNTID=$SSH_AUTH_SOCK`) when using BuildKit [docker/cli#1438](https://github.com/docker/cli/pull/1438) / [docker/cli#1419](https://github.com/docker/cli/pull/1419)
++ Add `--chown` flag support for `ADD` and `COPY` commands on Windows [moby/moby#35521](https://github.com/moby/moby/pull/35521)
++ Add `builder prune` subcommand to prune BuildKit build cache [docker/cli#1295](https://github.com/docker/cli/pull/1295) [docker/cli#1334](https://github.com/docker/cli/pull/1334)
```

Comment:
Hm... think I'm missing the BuildKit "garbage collect" changes in the changelog here

@tiborvass @tonistiigi other notable BuildKit changes we should include in the changelog?